demo/admin/src/products/ProductForm.gql.ts

```diff
@@ -0,0 +1,37 @@
+import { gql } from "@apollo/client";
+
+export const productFormFragment = gql`
+    fragment ProductFormFragment on Product {
```

Comment:
The fragment shouldn't have `Fragment` in its name, as this generates `GQLProductFormFragmentFragment` types